Lib9c/Action/DPoS/Sys/AllocateReward.cs

```diff
@@ -34,12 +36,14 @@ public override IWorld Execute(IActionContext context)
             var states = context.PreviousState;
             var nativeTokens = ImmutableHashSet.Create(
                 Asset.GovernanceToken, Asset.ConsensusToken, Asset.Share);
-            states = Control.AllocateRewardCtrl.Execute(
+            var previousProposerInfo =
```

Comment:
It looks like this code will throw an exception if the `AllocateReward` action is executed when the genesis block is created.